Blockzilla/BrowserViewController.swift

```diff
@@ -819,7 +820,22 @@ extension BrowserViewController: URLBarDelegate {
     func urlBar(_ urlBar: URLBar, didEnterText text: String) {
         // Hide find in page if the home view is displayed
         let isOnHomeView = homeView != nil
-        overlayView.setSearchQuery(query: text, animated: true, hideFindInPage: isOnHomeView)
+        if Settings.getToggle(.enableSearchSuggestions) {
+            searchSuggestClient.getSuggestions(text,callback: {suggestions, error in
+                if suggestions == nil {
+                    //self.overlayView.setSearchQuery(queryArray: [], animated: true, hideFindInPage: true)
+                    self.overlayView.setSearchQuery(query: text, animated: true, hideFindInPage: isOnHomeView)
+                } else {
+                    //self.overlayView.setSearchQuery(queryArray: suggestions!, animated: true, hideFindInPage: isOnHomeView)
+                    self.overlayView.setSearchQuery(query: text, animated: true, hideFindInPage: isOnHomeView)
```

Comment:
These if and else cases look to execute the same code. Is there something I'm missing here?

Reply:
They are because I made it so that the search suggestions were available but didn't want to include them in the OverlayView because doing so is a different issue.